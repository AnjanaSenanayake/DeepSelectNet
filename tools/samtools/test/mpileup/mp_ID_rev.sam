@SQ	SN:z	LN:13
@CO	TAG  CTTAG  CAGGT ref
@CO	 AG  CTTAGTT**GGC s1
@CO	 AG  CTTAGTT**    s2
@CO	 AG  CTTAG*T**    s3
@CO	   AA**TAG  CAGG  s4
@CO	   *A**TAG  CAGG  s5
@CO	 01AB23456CD78901
s1	16	z	2	0	7M2I2D3M	*	0	0	AGCTTAGTTGGC	0123456CD901
s2	16	z	2	0	7M2I2D	*	0	0	AGCTTAGTT	0123456CD
s3	16	z	2	0	7M1P1I2D	*	0	0	AGCTTAGT	0123456D
s4	16	z	4	0	2I2D7M	*	0	0	AATAGCAGG	AB4567890
s5	16	z	4	0	1P1I2D7M	*	0	0	ATAGCAGG	B4567890
