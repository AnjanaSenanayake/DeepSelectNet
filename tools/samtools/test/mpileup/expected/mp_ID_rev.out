z	2	A	3	^!,^!,^!,	000
z	3	G	3	,,,	111
z	4	C	5	,,,^!#^!#	22244
z	5	T	5	,,,##	33344
z	6	T	5	,,,,,	44444
z	7	A	5	,,,,,	55555
z	8	G	5	,+2tt-2ca,+2tt-2ca,+2#t-2ca,,	66666
z	9	C	5	###,,	9!!77
z	10	A	5	##$#$,,	9!!88
z	11	G	3	,,,	999
z	12	G	3	,,$,$	000
z	13	T	1	c$	1
